container/openejb-loader/src/main/java/org/apache/openejb/loader/FileUtils.java

```diff
@@ -120,10 +150,19 @@ public File getFile(final String path, final boolean validate) throws IOExceptio
         return file;
     }
 
+    /**
+     * Creates Temp Directory
+     * 
+     * Throws IOException if directory doesn't exists and if it fails to create directories
```

Comment:
- It belongs to `@throws` tag, I think.
- > if directory doesn't exists 
1. exist~~s~~
1. Which one? The one we want to create? We have no control over random used in the method, so how can programmer react to that IOEx?
- > doesn't exists and if it fails

What happens if all 100 *candidate directories* exist already? I see the method will throw as well. So your description is not quite correct.